examples/timeseries/timeseries_classification_from_scratch.py

```diff
@@ -0,0 +1,231 @@
+"""
+Title: Time series classification from scratch
+Author: [hfawaz](https://github.com/hfawaz/)
+Date created: 2020/07/21
+Last modified: 2020/07/21
+Description: Training a time series classifier from scratch on the Coffee dataset from the UCR/UEA archive.
+"""
+"""
+## Introduction
+
+This example shows how to do time series classification from scratch, starting from raw
+CSV time series files on disk, without leveraging pre-trained weights or a pre-made Keras
+Application model. We demonstrate the workflow on Coffee dataset from the [UCR/UEA
+dataset](https://www.cs.ucr.edu/%7Eeamonn/time_series_data_2018/).
+
+"""
+
+"""
+## Setup
+
+"""
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn import preprocessing
+
+"""
+## Load the data: the Coffee dataset
+The description of this dataset can be found
+[here](https://archive.uea.ac.uk/~ajb/Papers/BagnallSDM2012.pdf).
+
+### Read the TSV data online
+We will use the `Coffee_TRAIN` file for training and the
+`Coffee_TEST` file for testing. The simplicity of this dataset
+allows us to demonstrate effectively how to use ConvNets for time series classification.
+In this file, the first column corresponds to the label.
+"""
+
+
+def readucr(filename):
+    data = np.loadtxt(filename, delimiter="\t")
+    Y = data[:, 0]
+    X = data[:, 1:]
+    return X, Y
+
+
+root_url = "https://germain-forestier.info/dataset/UCRArchive_2018/Coffee/"
+
+x_train, y_train = readucr(root_url + "Coffee_TRAIN")
+x_test, y_test = readucr(root_url + "Coffee_TEST")
+
+"""
+## Visualize the data
+Here we visualize one time series example for each class in the dataset.
+
+"""
+
+classes = np.unique(np.concatenate((y_train, y_test), axis=0))
+
+plt.figure()
+for c in classes:
+    c_x_train = x_train[y_train == c]
+    plt.plot(c_x_train[0], label="class" + str(c))
+plt.legend(loc="best")
+plt.show()
+plt.close()
+
+"""
+## Standardizing the data
```

Comment:
Use preprocessing layers for this step https://keras.io/guides/preprocessing_layers/

Reply:
By looking at the Normalization preprocessing layer, it says: 
_The batch axis, 0, is always summed over (```axis=0``` is not allowed)._
However in this case, we want to apply znormalization to each time series independently, similar toInstanceNormalization.
So should I write a custom preprocessing layer for this one and include it in the file ?

Reply:
This comment should be addressed after solving [this](https://github.com/keras-team/keras-io/pull/159#discussion_r475484573).